review_practice/react-app/src/containers/SearchBar/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import Style from './style';
+
+import SearchButton from './SearchButton';
+import Logo from 'components/Logo';
+import Modal from 'components/Modal';
+import { DateFilter, PersonnelFilter, PriceFilter } from './ModalBody';
+
+import { isMobile } from 'Constants';
+import { clearToken } from 'service/TokenManager';
+
+export default ({ history }) => {
+    const dateButtonState = useState({ isClicked: false, buttonName: '날짜' });
+    const personnelButtonState = useState({ isClicked: false, buttonName: '인원' });
+    const priceButtonState = useState({ isClicked: false, buttonName: '가격' });
+    const buttonStateList = [dateButtonState, personnelButtonState, priceButtonState];
+    const buttonModalBodyList = [
+        (buttonState, setButtonState) => <DateFilter buttonState={buttonState} setButtonState={setButtonState} />,
+        (buttonState, setButtonState) => <PersonnelFilter buttonState={buttonState} setButtonState={setButtonState} />,
+        (buttonState, setButtonState) => <PriceFilter buttonState={buttonState} setButtonState={setButtonState} />,
+    ];
+
+    const searchStates = buttonStateList.map((state, index) => {
```

Comment:
p3;
`state[0]`, `state[1]` 이 무엇을 뜻하는지 바로 유추하기는 어려워 보입니다. (위쪽 코드를 한번 더 봐야하는.. ^^;;)
좀 더 나은 가독성을 위해 이렇게 하는 것은 어떨까요?

```suggestion
const searchStates = buttonStateList.map(([dateButtonState, personnelButtonState], index) => {
```